src/Infrastructure/Share/Migrations/Version20200727170306.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20200727170306 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
```

Comment:
Workaround for custom doctrine types that ends with never sync state due to comment issue